### DISCUSSION TITLE
error：error updating replication level for shard

### DISCUSSION THREAD

**Discussion by Johnite** [uber/cadence#4394] [Q&A] [answered]
cadence-server的日志中有这个error，{"level":"error","ts":"2021-08-20T11:47:09.264+0800","msg":"error updating replication level for shard","service":"cadence-history","shard-id":1054,"address":"IP:7934","component":"replication-ack-manager","error":"Failed to update shard. Previous range ID: 8401; new range ID: 8403","operation-result":"OperationFailed","logging-call-at":"task_ack_manager.go:205","stacktrace":"github.com/uber/cadence/common/log/loggerimpl.(*loggerImpl).Error\n\t/Users/didi/go/src/go.uber.org/cadence-0.20.0/common/log/loggerimpl/logger.go:131\ngithub.com/uber/cadence/service/history/replication.(*taskAckManagerImpl).GetTasks\n\t/Users/didi/go/src/go.uber.org/cadence-0.20.0/service/history/replication/task_ack_manager.go:205\ngithub.com/uber/cadence/service/history.(*historyEngineImpl).GetReplicationMessages\n\t/Users/didi/go/src/go.uber.org/cadence-0.20.0/service/history/historyEngine.go:2860\ngithub.com/uber/cadence/service/history.(*handlerImpl).GetReplicationMessages.func1\n\t/Users/didi/go/src/go.uber.org/cadence-0.20.0/service/history/handler.go:1565"}

这是什么原因呢？

**Comment by longquanzheng** [comment, marked as answer]
If not constantly happening, This is due to host restart/deployment and can be ignored 
https://github.com/uber/cadence/issues/4027

**Reply by Johnite** [reply]
那在重启之后，这种error会自动消失吗？

**Reply by longquanzheng** [reply]
The error should be gone after the hosts are stable. If not, then you probably have a critical error on your configuration like said in the issue—you probably running more than one clusters on the same database

**Reply by Johnite** [reply]
目前，只保留了一个active的cluster，其他的暂时没用
